src/api/app/models/bs_request.rb

```diff
@@ -295,7 +295,12 @@ def save!(args = {})
     new = created_at ? nil : 1
     sanitize! if new && !@skip_sanitize
     super
-    notify if new
+    if new
+      reviews.each do |review|
+        review.save!
```

Comment:
I'm pretty sure this is what `super` does. And I'm also pretty sure we have tests that create requests and test the notifications. So what you're saying is we miss tests to create requests with default reviewers?